packages/astro/test/0-css.test.js

```diff
@@ -8,260 +8,308 @@ import { expect } from 'chai';
-      const moduleClass = classes.find((name) => /^_title_[A-Za-z0-9-_]+/.test(name));
+      it('<style lang="scss">', async () => {
+        const el = $('#vue-scss');
 
-      // 1. check HTML
-      expect(el.attr('class')).to.include(moduleClass);
+        // 1. check HTML
+        expect(el.attr('class')).to.include('vue-scss');
 
-      // 2. check CSS
-      expect(bundledCSS).to.include(`${moduleClass}{`);
+        // 2. check CSS
+        expect(bundledCSS).to.match(new RegExp(`.vue-scss[^{]*{font-family:cursive`));
+      });
     });
 
-    it('<style lang="sass">', async () => {
-      const $ = index$;
-      const el = $('#vue-sass');
+    describe('Svelte', () => {
+      it('<style>', async () => {
+        const el = $('#svelte-css');
+        const classes = el.attr('class').split(' ');
+        const scopedClass = classes.find((name) => /^s-[A-Za-z0-9-]+/.test(name));
 
-      // 1. check HTML
-      expect(el.attr('class')).to.include('vue-sass');
+        // 1. check HTML
+        expect(el.attr('class')).to.include('svelte-css');
 
-      // 2. check CSS
-      expect(bundledCSS).to.match(new RegExp(`.vue-sass[^{]*{font-family:cursive`));
-    });
+        // 2. check CSS
+        expect(bundledCSS).to.match(new RegExp(`.svelte-css.${scopedClass}[^{]*{font-family:"Comic Sans MS"`));
+      });
+
+      it('<style lang="sass">', async () => {
+        const el = $('#svelte-sass');
+        const classes = el.attr('class').split(' ');
+        const scopedClass = classes.find((name) => /^s-[A-Za-z0-9-]+/.test(name));
 
-    it('<style lang="scss">', async () => {
-      const $ = index$;
-      const el = $('#vue-scss');
+        // 1. check HTML
+        expect(el.attr('class')).to.include('svelte-sass');
 
-      // 1. check HTML
-      expect(el.attr('class')).to.include('vue-scss');
+        // 2. check CSS
+        expect(bundledCSS).to.match(new RegExp(`.svelte-sass.${scopedClass}[^{]*{font-family:"Comic Sans MS"`));
+      });
 
-      // 2. check CSS
-      expect(bundledCSS).to.match(new RegExp(`.vue-scss[^{]*{font-family:cursive`));
+      it('<style lang="scss">', async () => {
+        const el = $('#svelte-scss');
+        const classes = el.attr('class').split(' ');
+        const scopedClass = classes.find((name) => /^s-[A-Za-z0-9-]+/.test(name));
+
+        // 1. check HTML
+        expect(el.attr('class')).to.include('svelte-scss');
+
+        // 2. check CSS
+        expect(bundledCSS).to.match(new RegExp(`.svelte-scss.${scopedClass}[^{]*{font-family:"Comic Sans MS"`));
+      });
     });
   });
 
-  describe('Svelte', () => {
-    it('<style>', async () => {
-      const $ = index$;
-      const el = $('#svelte-css');
-      const classes = el.attr('class').split(' ');
-      const scopedClass = classes.find((name) => /^s-[A-Za-z0-9-]+/.test(name));
+  // with "build" handling CSS checking, the dev tests are mostly testing the paths resolve in dev
+  describe('dev', () => {
```

Comment:
Dev server tests added. The problem was that the Windows URLs were confusingly resolving during build, but not dev.